ros_cross_compile/docker/build_workspace.sh

```diff
@@ -1,14 +1,10 @@
 #!/bin/bash
-# No -u because setup.bash for ROS has unset variables
-# No -x because it prints all of setup.bash steps, which are numerous and noisy
-set -eo pipefail
-
-[ "${ROS_DISTRO}" ] || (echo "ROSDISTRO var unset" && exit 1)
-[ "${TARGET_ARCH}" ] || (echo "TARGET_ARCH var unset" && exit 1)
-[ "${OWNER_USER}" ] || (echo "OWNER_USER var unset" && exit 1)
```

Comment:
Why were these lines removed?

Reply:
The set - u option raises an error on unset variables. We need them because we weren't setting it